backend/__tests__/cronJobs.test.ts

```diff
@@ -0,0 +1,122 @@
+import { expect } from 'chai';
+import mongoose from 'mongoose';
+import { MongoMemoryServer } from 'mongodb-memory-server';
+import ComplianceReport from '../src/db/schemas/compliance';
+import { startCronJobs } from '../src/cronJobs';
+import * as removeExpiredDrafts from '../src/cronJobs/removeExpiredDrafts';
+import { appConfig } from '../src/config';
+import sinon from 'sinon';
+
+describe('Cron Job', () => {
+    let mongod;
+    let clock;
+    let originalCronSchedule = appConfig.cron.removeExpiredDraftsSchedule;
+
+    before(async () => {
+        mongod = await MongoMemoryServer.create();
+        const uri = mongod.getUri();
+
+        if (mongoose.connection.readyState === 1) {
+            await mongoose.disconnect();
+        }
+        await mongoose.connect(uri);
+
+        const currentDate = new Date();
+        const expiredDate = new Date(currentDate);
+        expiredDate.setDate(expiredDate.getDate() - 1);
+        const futureDate = new Date(currentDate);
+        futureDate.setDate(futureDate.getDate() + 1);
```

Comment:
It's usually good approach to have fixed time values for this sort of testing. Even though we're using future and past day it's still not 100% error prone and could cause unexpected random fails in test pipeline.

Reply:
Good idea, changed.